SF iOS/SF iOS/AppDelegate.m

```diff
@@ -7,57 +7,34 @@
 
 @interface AppDelegate ()
 
-@property (nonatomic) EventDataSource *dataSource;
+@property (nonatomic) SwipableNavigationContainer *navigationContainer;
 
 @end
 
 @implementation AppDelegate
 
-- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-    self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-    [self.window makeKeyAndVisible];
-    
-    EventDataSource *datasource = [[EventDataSource alloc] initWithEventType:EventTypeSFCoffee];
-    EventsFeedViewController *feedController = [[EventsFeedViewController alloc] initWithDataSource:datasource];
-    
-    self.window.rootViewController = feedController;
-    
-    return true;
-}
-
-
-- (void)applicationWillResignActive:(UIApplication *)application {
-    // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-    // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-}
-
-
-- (void)applicationDidEnterBackground:(UIApplication *)application {
-    // Use this method to release shared resources, save user data, invalidate timers, and store enough application state information to restore your application to its current state in case it is terminated later.
-    // If your application supports background execution, this method is called instead of applicationWillTerminate: when the user quits.
+- (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    self.navigationContainer = [[SwipableNavigationContainer alloc] init];
+    return YES;
 }
 
-
-- (void)applicationWillEnterForeground:(UIApplication *)application {
-    // Called as part of the transition from the background to the active state; here you can undo many of the changes made on entering the background.
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [self.navigationContainer.window makeKeyAndVisible];
+    return YES;
 }
 
-
 - (void)applicationDidBecomeActive:(UIApplication *)application {
     // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
     [[NSNotificationCenter defaultCenter] postNotificationName:NSNotification.applicationBecameActiveNotification object:nil];
 }
 
-
-- (void)applicationWillTerminate:(UIApplication *)application {
-    // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+- (UIWindow *)window {
+    return self.navigationContainer.window;
 }
 
-
 @end
```

Comment:
I'm really happy that you even shortened up this size of this file. I know Jerry is doing some stuff in the AppDelegate as well. I'll need to make sure his stuff is compatible with this changes too. https://github.com/ThumbWorks/sf-ios/compare/master...jerrytdev:local-notifications